models/bert/__main__.py

```diff
@@ -93,7 +99,7 @@ def evaluate_split(model, processor, args, split='dev'):
 
     train_examples = None
     num_train_optimization_steps = None
-    if args.do_train:
+    if args.trained_model:
```

Comment:
This doesn't sound right. Did you try running this with and without a pre-trained model? processor.get_train_examples should execute even when we don't have a pre-trained model

Reply:
Sorry I am retracting the last commit. I am not having this change in this PR

Reply:
Alright can you merge it now!